docs/configuration.asciidoc

```diff
@@ -0,0 +1,131 @@
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[configuration]]
+== Configure
+
+:language: Java
+:language_lc: java
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+////
+TO DO: How do users set configuration options?
+////
+[discrete]
+[[configure-methods]]
+=== Configuration methods
+
+////
+TO DO:
+Are there multiple ways to set configuration options?
+For example, can they use environment variables OR a config file?
+If there are, what are they and why would you use each method?
+Where/how do you use each method?
+Provide an example to illustrate the syntax.
+////
+
+////
+TO DO:
+If configuration options are defined using multiple methods,
+which one takes precedence over the other(s)?
+////
+
+[discrete]
+[[configuration-options]]
+=== Configuration options
+
+Because the Elastic Distribution for OpenTelemetry Java ("the distro") is an extension of the https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry Java agent], it supports both:
+
+* General OpenTelemetry SDK configuration options
+* Elastic-specific configuration options that are only available when using the distro
+
+[discrete]
+[[configure-otel-sdk-options]]
+==== OpenTelemetry SDK configuration options
+
+////
+TO DO:
+Is this true?
+Are there any options that aren't supported?
+Any options that shouldn't be used?
+////
+The distro supports all configuration options listed in the https://opentelemetry.io/docs/languages/sdk-configuration/general/[OpenTelemetry General SDK Configuration documentation].
+
+////
+TO DO:
+Are there Elastic-specific custom configuration options
+in addition to the general OpenTelemetry SDK
+configuration options? Or are all the customizations
+"behind the scenes"?
+If not, you can delete this section.
+////
+[discrete]
+[[configure-distro-options]]
+==== Elastic-specific configuration options
+
+The distro supports the following Elastic-specific options:
+
+////
+List config options
+////
+
+////
+TO DO:
+Are there multiple authentication methods when sending data to Elastic?
+If not, delete this section.
+////
+[discrete]
+[[configure-auth-methods]]
+=== Authentication methods
+
+When sending data to Elastic, there are two ways you can authenticate: using a secret token or using an APM agent key.
+
+[discrete]
+[[configure-secret-token]]
+==== Use a secret token
+
+// TO DO: What is this?
+// ??
+
+// TO DO:Why would you choose this method?
+// ??
+
+// TO DO:How do you authenticate using this method?
+// ??
+
+[discrete]
+[[configure-api-key]]
+==== Use an APM agent key (API key)
+
+// ✅ What is this?
+It is also possible to authenticate to an Elastic Observability endpoint using
+an _APM agent key_. These are revocable API keys.
+
+// TO DO: Why would you choose this method?
```

Comment:
A user would choose an API key over a secret token because:

* you can have multiple api keys
* you can add/remove api keys without restarting APM Server

Reply:
this info should be added in the apm guide too